docs/guide/validations.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: default
+title: Validations
+parent: Building ViewComponents
+---
+
+# Validations
```

Comment:
Somewhat related to my top-level comment, but this seems more like an FAQ entry or a recipe, thoughts on removing it for now in favor of bringing it back in some other, less official form?

Reply:
Moved to FAQs 👍🏻